pom.xml

```diff
@@ -28,7 +28,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-      <version>1.447</version>
+      <version>1.554.1</version>  <!-- What version does maintainer want here? I added to get @DataBoundSetter -->
```

Comment:
@tfennelly this is also the source of your lovely test failures.  I'm not sure exactly why thats the case but updating the jenkins core version is what brough this upon you.